### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by tigamadou**
- Created the directory structure - components- containers - actions - reducers 
- Implemented the App component  rendering BooksList and BooksForm
- Implemented the BooksList component
- Implemented the Book component
- Implemented the BooksForm component
- Defined the Redux actions and reducers
- Initialized the Redux store

### CODE REVIEW

**Comment by BrittanyBlake** [comment]
### Status: Invalid 🛑 
****
Hi @tigamadou. This pull request has been marked as invalid because you did not add submit for a review with a descriptive [Pull Request](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/).

- Please ensure that your PR has a summary of what you have done in this milestone before resubmitting. 
****
Please make the required adjustments and submit a new code review request.

### Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({
+  id, title, category,
+}) => (
```
Comment (BrittanyBlake) [inline src/components/Book.js]: - [x] The Book component should accept the `book` object as a prop.
 * You don't have to break down the book properties at the prop level. You can just pass the book object, which will be a representation of the book attributes (id, title, category)

File: src/actions/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+};
+export const createBook = (book = defaultBook) => ({
+  type: ADD_BOOK,
+  book,
+});
+
+export const removeBook = ({ id }) => ({
+  type: REMOVE_BOOK,
+  id,
+});
```
Comment (BrittanyBlake) [inline src/actions/index.js]: - [x] The `REMOVE_BOOK` action should use the **book object** as an argument, not the id.
Screenshot from pathwright 👇🏾 
Screen Shot 2021-04-15 at 9 56 17 PM

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi Team! 👋 
 You have done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****
* Please see my comments below for details.

****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

File: src/components/Book.js (hunk 1 above)
Comment (tigamadou) [reply src/components/Book.js]: 👍

File: src/actions/index.js (hunk 2 above)
Comment (tigamadou) [reply src/actions/index.js]: 👍

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations Amadou :tada: :clinking_glasses: 

**This milestone has passed all the mandatory requirements :clap: 
It is therefore approved :heavy_check_mark:**

**Comment(s)**

Against your next milestone:
- Kindly remove the `.vscode` directory in your project. This folder is not related to the project and therefore, it should not be committed.
- You should also remove the `logo.svg` image.

Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Comment by tigamadou** [comment]
@BrittanyBlake @teekaytech Thank you very much for your hard work. 👍

**Pull request merged**